Library/PhotosController.swift

```diff
@@ -48,9 +47,31 @@ class PhotosController: UICollectionViewController {
             self.sections = Photo.constructRemoteElements()
             self.collectionView?.reloadData()
         }
+
+        #if os(tvOS)
+        let playPauseTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.playPause(gesture:)))
+        playPauseTapRecognizer.allowedPressTypes = [NSNumber(value: UIPressType.playPause.rawValue)]
+        self.collectionView?.addGestureRecognizer(playPauseTapRecognizer)
+        #endif
     }
 
     #if os(tvOS)
+    func playPause(gesture: UITapGestureRecognizer) {
```

Comment:
There's no pause for a slideshow, you press "play" for starting the slideshow and press "Menu" (it means Home/Back in the Apple TV) to stop the slideshow.